db/models/score/scores.py

```diff
@@ -0,0 +1,41 @@
+"""The module containing all code related to the `scores` table
+within the Postgres db.
+"""
+import uuid
+from db import db
+from sqlalchemy import ForeignKey
+from sqlalchemy.sql import func
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class Score(db.Model):
+    """Score The sqlalchemy-flask model class used to define the
+    `scores` table in the Postgres database."""
+
+    __tablename__ = "scores"
+
+    id = db.Column(
+        "score_id", UUID(as_uuid=True), default=uuid.uuid4, primary_key=True
+    )
```

Comment:
Using `sqlalchemy`s Column has better autocomplete and doesn't change the migrations.